src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -649,7 +648,7 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
       const id = person?.officer?.id
 
       let thisPerson
-      if (isNaN(id)) {
+      if (isNaN(+id)) {
```

Comment:
I guess the code was expecting id to be a number (not a string). Anyway, this works if string is null or empty, ie,

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/2cb2b3da-25bd-4263-91fd-2ea09ba814c9)